bitnami/kafka/templates/statefulset.yaml

```diff
@@ -188,6 +189,12 @@ spec:
               value: {{ .Values.listenerSecurityProtocolMap | quote }}
               {{- else if .Values.externalAccess.enabled }}
               value: "INTERNAL:{{ $interBrokerProtocol }},CLIENT:{{ $clientProtocol }},EXTERNAL:{{ $externalClientProtocol }}"
+              {{- else if .Values.kraft.enabled }}
+                {{- if .Values.externalAccess.enabled }}
```

Comment:
Something is wrong here. A previous branch in the if also checks if .Values.externalAccess.enabled is true doesn't make sense to test it here again

Reply:
fixed it